packages/flutter/lib/src/material/chip.dart

```diff
@@ -1821,6 +1821,10 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     assert(debugCheckHasDirectionality(context));
     assert(debugCheckHasMaterialLocalizations(context));
 
+    /// Now the chip starts with 8px on each side and as text scaling gets
+    /// closer to 2 the label padding is linearly interpolated from 8px to 4px.
+    /// Once the widget has a text scaling of 2 or higher than the label padding
```

Comment:
2 or higher => 2 or lower

Reply:
It should remain 2 or higher because the widget will have 4px label padding only if the text scaling is 2 or higher.

Reply:
What happens when the text scale factor drops below 2? That seemed to be the point you were trying to make here, since the first sentence describes what happens as the text scale factor decreases towards 2.

Reply:
The chip starts with 8px on each side and as text scaling gets closer to 2 the label padding is linearly interpolated from 8px to 4px. 

If they text scaling decreases towards 2. Lets say text scaling is set to 3 and if we change it to 2 than the label padding remains 4px.

Reply:
The chip at text scale 1 starts with 8px on each side and as text scaling gets closer to 2 the label padding is linearly interpolated from 8px to 4px.

Reply:
Thanks for updating the comment, that's better.

Reply:
Done